content/upgrading-tickets.md

```diff
@@ -0,0 +1,20 @@
+type: page
+title: Upgrading your ticket
+url: tickets/upgrading/index.html
+tito_required: yes
+---
+
+If you have already bought a ticket,
+but would like to upgrade it (perhaps from a 3-day ticket to a 4-day ticket,
+or from a self-funded ticket to an employer-funded ticket),
+you can purchase an upgrade.
+
+Within the limits of our ticketing system,
+the simplest way for us to allow you to do this has been to create a special ticket with a "pay what you want" price field.
+Of course, you shoudn't actually pay us what you want!
+You should calculate the difference between the price of your original ticket and the price of your new ticket.
```

Comment:
Can we give them a table of upgrades, or some such such? Or just ask them to ask us?